### DISCUSSION TITLE
wgml.exe: why is there no native version?

### DISCUSSION THREAD

**Discussion by LowLevelMahn** [open-watcom/open-watcom-v2#1344] [General]
so x86/x64 win/linux

just never tried to port it or bad expirience trying to?

(wgml takes a good while to run - a native port could maybe speedup the build a little)

what is needed to build the DOS version from source (and run the tests)?

**Comment by jmalak** [comment]
We have no source code only binary version.
on the original openwatcom.org Perforce server there was a project to create a wgml source code from scratch, but now it is no longer available, the latest version is in the OW 1.9 repository, but it is unfinished. I made a port to OW2.0 on some stage, but it is not yet fully functional, so it is not possible to replace existing wgml program with this version.

**Comment by LowLevelMahn** [comment]
>We have no source code only binary version.

thats really bad - and a big blocker for just porting :(

**Reply by jmalak** [reply]
Exactly
